applications/error-notifications.md

```diff
@@ -0,0 +1,32 @@
+# Error Notifications
+
+In order to alert teams to server errors for applications we need a way of
```

Comment:
I'd probably come at this as more of a statement, and then gives the reasons why, for example

> Applications should report errors in an [Airbrake](https://airbrake.io/) compliant format to an internally maintained instance of [Errbit](https://github.com/errbit/errbit) (details available from team leads).
>
> Reporting them in this way enables
> - **Easier access for teams**; no need to request permission to access production logs
> - **More details**; Airbrake compliant errors often will include the full stack trace along with meaningful data about the environment at the time
> - **Less noise**; you get just the error rather than trying to extract it from other output in the log, and Errbit will automatically group errors
> - **Better integrations**; we can integrate Errbit with the tools we use like [Slack](https://slack.com/)
>
> Below are examples of how to integrate Airbrake into the your app, based on technology.

What do you think?